src/EventSubscriber/HttpCacheResponseSubscriber.php

```diff
@@ -39,9 +39,13 @@ public static function getSubscribedEvents()
 
     public function configureCache(FilterResponseEvent $event)
     {
-        $view = $event->getRequest()->attributes->get('view');
-        if (!$view instanceof CachableView || !$view->isCacheEnabled()) {
-            return;
+        if ($event->getRequest()->attributes->has('is_rest_request') && $event->getRequest()->attributes->get('is_rest_request') === true) {
```

Comment:
I don't see why this needs to be added, afaik REST already sets `s-maxage` doesn't it? Is this needed to solve this bug or can this be handled separately?

Reply:
Yes, that is required because for REST requests the kernel does not return a view.
Then, as it is not a view, it returns `null` and nothing is tagged accordingly.
Please note that when we return null here, the response is cached for the default time.